src/lib/parameters/param_translation.cpp

```diff
@@ -146,6 +146,25 @@ bool param_modify_on_import(bson_node_t node)
 		}
 	}
 
+	// 2022-04-11: translate VT_PTCH_MIN to VT_PITCH_MIN
+	{
+		if (strcmp("VT_PTCH_MIN", node->name) == 0) {
+			strcpy(node->name, "VT_PITCH_MIN");
+			PX4_INFO("copying %s -> %s", "VT_PTCH_MIN", "VT_PITCH_MIN");
+			return true;
+		}
+	}
+
+	// 2022-04-11: translate VT_LND_PTCH_MIN to VT_LND_PITCH_MIN
+	{
+		if (strcmp("VT_LND_PTCH_MIN", node->name) == 0) {
+			strcpy(node->name, "VT_LND_PITCH_MIN");
```

Comment:
@dagar What do we have as limit for parameter names nowdays. I remember it used to be 16 characters including the null termination?

Reply:
The limit is 16 characters not including NULL termination, but in some areas like mavlink you have to be careful because the storage might be char[16] (only terminated by NULL if the length is less than 16).

Reply:
So I leave it at 16 I guess.